inlang/documentation/ecosystem/overview/introduction.md

```diff
@@ -1,6 +1,59 @@
-# Under construction
+# Introduction
 
-See old docs start page [here](/documentation/sdk)
+Welcome to the inlang documentation!
+
+---
+
+## What is inlang?
+
+![ecosystem illustration](https://cdn.jsdelivr.net/gh/opral/monorepo@latest/inlang/documentation/ecosystem/assets/ecosystem-atom.jpg)
+
+<br>
+
+inlang offers tailored localization products for each [use case](/#personas) (Developer, Designer, Translator). You only have to set inlang up once; then, you can use every inlang app out of the box. 
+
+Some of the main inlang features include:
+
+- **Decentralized workflows** - Don't force devs in a cloud solution; let devs, translator and designer work in their native environment. ([Apps](/c/apps))
+- **One source of truth** - Never have syncing problems again; The data is always stored in your repo while collaborator can work on it in their own interface.
+- **Quality management** - Define [Lint Rules](/documentation/lint-rule) for your project to ship quality cross-app. 
+- **Automations** - With lix change control you can use CI/CD to check lint, machine translate, validate and more. (Continuous Localization)
+- **Review** - PR workflows can be used to gain control over the merged content.
+- **Integration** - With [plugins](/c/plugins) every project can get inlang ecosystem compatible.
```

Comment:
How about
> plugins make every project compatible with the inlang ecosystem.

Reply:
Project is ambiguous. Inlang project? Software project? 

> Plugins - Adapt the inlang ecosystem with plugins to your needs, such as integrations to external services.